src/main/resources/twilio-python/version.mustache

```diff
@@ -0,0 +1,31 @@
+{{>licenseInfo}}
+from twilio.base.version import Version
+from twilio.base.domain import Domain
+{{#versionResources}}from twilio.rest.{{domainPackage}}.{{apiVersion}}.{{filename}} import {{importName}}
+{{/versionResources}}
+
+class {{apiVersionClass}}(Version):
+
+    def __init__(self, domain: Domain):
+        """
+        Initialize the {{apiVersionClass}} version of {{domainPackage}}
+
+        :param domain: The Twilio.{{domainPackage}} domain
+        """
+        super().__init__(domain)
```

Comment:
I think the version and `self` should still be getting included here, no? ([example](https://github.com/twilio/twilio-python/blob/8.0.0-rc/twilio/rest/flex_api/v2/__init__.py#L22))

Reply:
The syntax changed in Python3 so you can just call it like this now. https://docs.python.org/3/library/functions.html#super